Whereabouts removal (#1748)
Mellanox/network-operator: example/README.md

...
# kubectl apply -f ./crs/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml
```

> Deploy a kubernetes secondary network from `networking` folder, there are `macvlan` and `host-device` networks each has example file for deploying with/without NVIDIA IPAM plugin

Reviewer: ```suggestion
Deploy a kubernetes secondary network from `networking` folder, there are example files for `macvlan` and `host-device` networks to deploy them with/without NVIDIA IPAM plugin
```
